src/Control/Carrier/Throw/Either.hs

```diff
@@ -1,4 +1,8 @@
 {-# LANGUAGE FlexibleInstances, GeneralizedNewtypeDeriving, MultiParamTypeClasses, TypeOperators, UndecidableInstances #-}
+
+-- | An 'Either'-based carrier for the 'Throw' effect.
```

Comment:
Added this, since we forgot it.

Reply:
(Heads up that this conflicts with #275.)